static/sass/layout/_nav.scss

```diff
@@ -1,23 +1,100 @@
 // Site Navigation
 // ===============
 
-$nav-locations: 'birds', 'work', 'speaking', 'open-source', 'blog', 'community', 'contact';
+$locations: 'projects', 'open-source', 'blog', 'birds', 'speaking', 'community', 'contact';
+$nav-locations: 'work', 'about';
 
 .nav-link {
+  color: color('action-alt');
+  display: inline-block;
+  font-weight: bold;
+  padding: size('half-shim') size('shim');
+  text-decoration: none;
+  white-space: nowrap;
+
+  @include above('nav-narrow') {
+    padding: size('shim') size('gutter');
+  }
+
   #{$link} {
-    display: inline-block;
-    padding: size('half-shim');
-    text-decoration: none;
+    color: color('action-alt');
+  }
+
+  #{$focus} {
+    color: color('action');
+  }
+
+  &[aria-pressed='true'] {
+    color: color('action');
   }
 }
 
-@each $location in $nav-locations {
+@each $location in $locations {
   [data-location='#{$location}'] {
     [data-active='#{$location}'] {
-      #{$link} {
-        color: color('action');
-        font-style: italic;
-      }
+      color: color('action');
+    }
+  }
+}
+
+@each $location in $nav-locations {
```

Comment:
I don't quite understand how that would work. The two loops are using different selectors, `[data-location]` and `[data-nav-location]`. I guess it would have to look like:

```scss
$parent-nav: (
  'projects': 'work', 
  'open-source': 'work', 
  'birds': 'about', 
  'speaking': 'about', 
  'community': 'about', 
);
@for $location, $parent in $parent-nav {
  [data-location='#{$location}'] {
    [data-active='#{$location}'] {
      color: color('action');
    }
  }

  [data-nav-location='#{$location}'] {
    [data-active='#{$parent}'] {
      color: color('action');
    }
  }
}
```